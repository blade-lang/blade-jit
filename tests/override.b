class A {
  @abs() {
    return 300
  }

  to_string() {
    return 'A class called A'
  }
}

echo abs(-10)
echo abs(new A())
echo new A().to_string()
