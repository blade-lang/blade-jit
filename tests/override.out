10
300
A class called A